api/src/main/java/edu/wpi/first/shuffleboard/api/util/TypeUtils.java

```diff
@@ -24,4 +25,37 @@ private TypeUtils() {
   public static <T> Function<Object, Optional<T>> optionalCast(Class<T> cls) {
     return value -> optionalCast(value, cls);
   }
+
+  /**
+   * Allows you to filter out members of a subtype from a stream of some base type.
```

Comment:
Usually if documentation uses the word 'you' it could be improved open to make the documentation clearer.